ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__cr-4.16-nightly.yaml

```diff
@@ -24,7 +24,7 @@ resources:
       memory: 200Mi
 tests:
 - as: krkn-hub-aws
-  cron: 0 0 * * 0,3
```

Comment:
with 4.16 being our fourth supported version, should we put this to only twice a week?

Reply:
Done!